src/components/side_panel/carousel_panel/carousel_panel.ts

```diff
@@ -148,6 +145,44 @@ export class CarouselPanel extends Component<Props, SpreadsheetChildEnv> {
     return getCarouselItemPreview(this.env.model.getters, item);
   }
 
+  updateItems(items: CarouselItem[]) {
+    this.env.model.dispatch("UPDATE_CAROUSEL", {
+      figureId: this.props.figureId,
+      sheetId: this.env.model.getters.getActiveSheetId(),
+      definition: { ...this.carousel, items },
+    });
+  }
+
+  updateTitleText(title: string) {
```

Comment:
nitpick: either name it updateCarouselTitleText or rename the other method updateTitleStyle for consistency